.circleci/config.yml

```diff
@@ -71,27 +75,20 @@ executors:
   ci-images:
     docker:
       - image: quay.io/reactiveops/ci-images:v11-stretch
-  ci-images-large:
+  goreleaser:
     resource_class: large
```

Comment:
```suggestion
```

Reply:
@rbren It looks like you're suggesting to remove the large resource-class - we may need this to help goreleaser have CPU/memory to build all of the plugins.

Reply:
I added the large resource class a while ago to help speed up the build. It's possible we still need it, but let's see how it runs without

Reply:
(we can do that in a follow-up PR if you prefer)

Reply:
@rbren I put this executor size back because the builds ran out of memory.